RockPaperScissors.rb

```diff
@@ -0,0 +1,65 @@
+=begin
+Abigail and Benson are playing Rock, Paper, Scissors.
+
+Each game is represented by an array of length 2, where the first element represents what Abigail played and the second element represents what Benson played.
+
+Given a sequence of games, determine who wins the most number of matches. If they tie, output "Tie".
+
+R stands for Rock
+P stands for Paper
+S stands for Scissors
+
+Examples
+calculate_score([["R", "P"], ["R", "S"], ["S", "P"]]) ➞ "Abigail"
+
+# Ben wins the first game (Paper beats Rock).
+# Abigail wins the second game (Rock beats Scissors).
+# Abigail wins the third game (Scissors beats Paper). 
+# Abigail wins 2/3.
+
+calculate_score([["R", "R"], ["S", "S"]]) ➞ "Tie"
+
+calculate_score([["S", "R"], ["R", "S"], ["R", "R"]]) ➞ "Tie"
+=end
+
+def calculate_score(array)
```

Comment:
better algorithm suggestion:

what if you had a new method called `who_wins(array)`. The array you pass in could be like `["R", "P"]`. The method then figures out who wins this. So it will return either `0` or `1` for who won this. 

Then from where you are calling this from, you can keep track of who won all the three. And print out who won or if it is a tie.

Reply:
The algorithm worked for me thanks